Tests/test_Nexus.py

```diff
@@ -494,6 +494,42 @@ def test_TreeTest2(self):
         tree.collapse_genera()
         self.assertEqual(tree.all_ids(), [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 13, 14, 17])
 
+    def test_TreeTest3(self):
+        """Test merge_withSupport and consensus method."""
```

Comment:
Typo, ``merge_withSupport`` --> ``merge_with_support``.

Does this indirectly test the consensus method?

Reply:
Yes it does.